internal/cmd/envs.go

```diff
@@ -62,8 +62,115 @@ func envsCommand() *cobra.Command {
 	}
 	lsCmd.Flags().StringVarP(&outputFmt, "output", "o", "human", "human | json")
 	cmd.AddCommand(lsCmd)
-	cmd.AddCommand(stopEnvCommand(&user))
 
+	var (
+		img      string
+		tag      string
+		cpuCores float32
+		memGB    float32
+		diskGB   int
+		gpus     int
+		services []string
+	)
+
+	editEnvCmd := &cobra.Command{
```

Comment:
Can we move this into its own function? The `lsCmd` is total garbage and is getting fixed as part of https://github.com/cdr/coder-cli/pull/145